composer.lock

```diff
@@ -964,10 +965,10 @@
         },
         {
             "name": "koodimonni-language/core-en_gb",
-            "version": "5.9.3",
+            "version": "6.0.1",
```

Comment:
Hey Meena :) I notice the core language has been updated for WordPress; however, the core WP installation hasn't been updated. This constraint is managed in `composer.json` - it would be worth checking that WP can be updated to version 6.0.1 and that the WP language core and core versions match.